angular/src/app/customer-chat/customer-chat.component.html

```diff
@@ -0,0 +1,200 @@
+<div id="card-body-chat">
+  <div id="video-wrapper" *ngIf="localVideoStream || remoteVideoStream">
```

Comment:
Würde ich in eine eigene Video-Component auslagern

Reply:
Ok die Date erscheint mir allgemein noch relativ groß und sollte sich denke ich noch gut in kleinere Komponenten unterteilen lassen

Reply:
Habe in a023b53e5f1dac67699cfd2d0c9d1c0cbff1f9f2 die Anwendung noch weiter unterteilt. Passt das so?